packages/react-accordion/Spec.md

```diff
@@ -388,6 +396,9 @@ Expected DOM output
 
 ## Behaviors
 
+- Arrow Navigation should be optional and disabled by default.
+- Circular Navigation should be optional and disabled by default.
```

Comment:
Do you have the keyboard navigation specced here ? I don't think we should give consumers a choice for this if the WAI spec requires it

```suggestion
- Circular Navigation should be optional and disabled by default.
```

Reply:
This has been extensively discussed already

The conclusion is: Implement both navigation and cyclical but disable both by default.
The WAI spec doesn't require it also, it's optional, and the argument in favor of making it disabled by default are quite strong, I'll see to make the Spec more clear on that!